packages/agents-hosting/src/oauth/userTokenClient.ts

```diff
@@ -15,14 +15,12 @@ const logger = debug('agents:user-token-client')
  */
 export class UserTokenClient {
   client: AxiosInstance
-  msAppId: string
   /**
    * Creates a new instance of UserTokenClient.
    * @param token The token to use for authentication.
    * @param msAppId The Microsoft application ID.
    */
-  constructor (token: string, appId: string) {
-    this.msAppId = appId
+  constructor (private msAppId: string) {
```

Comment:
token out of the ctor, so the caller needs to set it before invoking any method.